cli-plugins/plugin/plugin.go

```diff
@@ -131,7 +131,7 @@ func newPluginCommand(dockerCli *command.DockerCli, plugin *cobra.Command, meta
 			DisableDescriptions: true,
 		},
 	}
-	opts, flags := cli.SetupPluginRootCommand(cmd)
+	opts, _ := cli.SetupPluginRootCommand(cmd)
```

Comment:
FWIW; it could also be removed from `SetupPluginRootCommand()` but that one has external consumers, so I didn't want to break them in this PR 😅